opsdroid/cli/utils.py

```diff
@@ -270,3 +270,31 @@ def build_config(ctx, params, value):
 
             click.echo(click.style("SUCCESS:", bg="green", bold=True), nl=False)
             click.echo(" Opsdroid modules successfully built from config.")
+
+
+def clear_logs(ctx, params, value):
+    """Clear all logs.
+
+    Asks for the user confirmation if the logs should be cleared or not. If user chooses
+    yes then all logs will be clear otherwise the action will be aborted.
+
+
+    Args:
+        ctx (:obj:`click.Context`): The current click cli context.
+        params (dict): a dictionary of all parameters pass to the click
+            context when invoking this function as a callback.
+        value (string): the value of this parameter after invocation.
+            It is either "config" or "log" depending on the program
+            calling this function.
+
+    """
+    click.echo(click.style("WARNING:", bg="yellow", bold=True), nl=False)
+    click.confirm(
+        " Are you sure you wish to clear all your logs? This action cannot be undone.",
+        abort=True,
+    )
+
+    open(DEFAULT_LOG_FILENAME, "w").close()
```

Comment:
I think the logs location is configurable.

Reply:
Good point I was looking at the edit cli function. This made me realise that the command would only work if the log was using the default location as well. 

With this in mind should we handle the case where there are more than one log file in the logs folder? 🤔

Reply:
Maybe rather than adding these CLI options it would be better for opsdroid to rotate its own logs?

http://www.blog.pythonlibrary.org/2014/02/11/python-how-to-create-rotating-logs/

Reply:
Yeah that will be a better option, I'm going to start working on that. But since the edit cli doesn't work on logs that don't use the default name I'll use this PR to fix that and then raise a new one adding rotation 😄